src/multibody/data.cpp

```diff
@@ -0,0 +1,5 @@
+#include "data.hpp"
```

Comment:
Here, according to https://en.cppreference.com/w/cpp/preprocessor/include, I don't know how much the directive might be well respected by the compiler. Maybe it would be better to use the full path from the pinocchio directory.

Reply:
I am surprised the standard is so vague on this topic. Everything is implementation defined...